long_tests/test_deep_training.py

```diff
@@ -98,18 +99,18 @@ def test_mmdetection():
         first_region = post_model_outputs_response.outputs[0].data.regions[0]
 
         bounding_box = first_region.region_info.bounding_box
-        np.testing.assert_allclose([bounding_box.top_row, bounding_box.left_col, bounding_box.bottom_row, bounding_box.right_col], [0.2178, 0.1606, 0.9614, 0.7952], rtol=0.05, atol=0.05)
+        assert calculate_iou([bounding_box.top_row, bounding_box.left_col, bounding_box.bottom_row, bounding_box.right_col], [0.2289, 0.1643, 1, 0.8023]) > .9, str(post_model_outputs_response)
 
-        assert len(first_region.data.concepts), 1
+        assert len(first_region.data.concepts) == 1, str(post_model_outputs_response)
         first_region_concept = first_region.data.concepts[0]
-        assert first_region_concept.id == "18"
-        assert first_region_concept.name == "dog"
-        assert first_region_concept.value > 0.5
+        assert first_region_concept.id == "18", str(post_model_outputs_response)
+        assert first_region_concept.name == "dog", str(post_model_outputs_response)
+        assert first_region_concept.value > 0.5, str(post_model_outputs_response)
```

Comment:
If we are using pytest to run these tests we could just skip adding that message to assert. 

We already enabled `--showlocals` by default which will print local variables state.   `-vv`  is also enabled, so it will print the whole value.  These options are present in `pytest.ini`.

Reply:
updated